iOS-remote-node-setup.md

```diff
@@ -71,7 +71,7 @@ export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
 Just to make sure everything went ok we can run `go version` to ensure we have go 1.10 installed:
```

Comment:
Should say "to ensure we have go 1.11 installed"

Reply:
Fixed @mrfelton